controllers/expose/ingressClassCreator.go

```diff
@@ -88,12 +83,7 @@ func (icc ingressClassCreator) CreateIngressClass(ctx context.Context) error {
 
 // isIngressClassAvailable check whether an ingress class with the given name exists in the current namespace.
```

Comment:
Is only the namespace really checked? It seems to me that it is a cluster-wide search for IngressClasses with the name